packages/modules/providers/notification-stdout/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "lib": ["es2020"],
+    "target": "es2020",
+    "jsx": "react-jsx" /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */,
+    "outDir": "../notification-console/dist",
+    "esModuleInterop": true,
+    "declaration": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "noImplicitReturns": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
+    "noImplicitThis": true,
+    "allowJs": true,
+    "skipLibCheck": true,
+    "downlevelIteration": true, // to use ES5 specific tooling
+    "inlineSourceMap": true /* Emit a single file with source maps instead of having a separate file. */
+  },
+  "include": ["../notification-console/src"],
+  "exclude": [
+    "../notification-console/dist",
+    "../notification-console/build",
+    "../notification-console/src/**/__tests__",
+    "../notification-console/src/**/__mocks__",
+    "../notification-console/src/**/__fixtures__",
+    "../notification-console/node_modules",
+    "../notification-console/.eslintrc.js"
```

Comment:
why is this checking another package? 🤔

Reply:
@riqwan nice catch, I just renamed the package and something must have happened with my git, this needs to be removed (done!)